test.py

```diff
@@ -241,9 +245,9 @@ async def add_test(test_file) -> None:
                     await self.add_test(test_file)
                     self.pending_test_count += 1
 
-            for p in patterns:
-                if p in t:
-                    add_test_tasks.append(asyncio.create_task(add_test(test_file)))
+            if not is_disabled(test_file) and not should_skip(t) and is_enabled(t):
```

Comment:
How do I know when I should pass `t` and when should I pass `test_file`? Each of these 3 functions are taking `name: str` param, which suggests they are expecting the same thing, but it turns out that `is_disabled` is subtly different from the other two...

Reply:
Fixed in https://github.com/scylladb/scylladb/commit/9e1eb6bbcec97e76abdbc7933dd1d1468811453a